# Define the goal
float32 goal_angle  # Specify which dishwasher we want to use
---
# Define the result
uint32 goal_done
---
# Define a feedback message
float32 angle_feed